spec.html

```diff
@@ -6410,20 +6409,22 @@ <h1>ValidateAndApplyPropertyDescriptor (_O_, _P_, _extensible_, _Desc_, _current
               1. If _O_ is not *undefined*, convert the property named _P_ of object _O_ from an accessor property to a data property. Preserve the existing values of the converted property's [[Configurable]] and [[Enumerable]] attributes and set the rest of the property's attributes to their default values.
           1. Else if IsDataDescriptor(_current_) and IsDataDescriptor(_Desc_) are both *true*, then
             1. If the [[Configurable]] field of _current_ is *false*, then
-              1. Return *false*, if the [[Writable]] field of _current_ is *false* and the [[Writable]] field of _Desc_ is *true*.
               1. If the [[Writable]] field of _current_ is *false*, then
+                1. Return *false*, if the [[Writable]] field of _Desc_ is present and the [[Writable]] field of _Desc_ is *true*.
                 1. Return *false*, if the [[Value]] field of _Desc_ is present and SameValue(_Desc_.[[Value]], _current_.[[Value]]) is *false*.
+                1. Return *true*.
```

Comment:
it seems like this change actually bypasses *applying* the Property Descriptor, in the final step. I think this line should not have been added.